HACKING.md

```diff
@@ -17,6 +17,26 @@ On Debian/Ubuntu:
 
     $ sudo apt install nodejs
 
+
+Creating VM images locally (not necessary for running tests) needs the
+following:
+    
+    $ sudo apt-get install virtinst
+
+For running integration tests, the following dependencies are required:
+
+    $ sudo apt-get install curl expect xz-utils rpm libvirt-daemon-system libvirt-daemon-driver-storage-core \
+        libvirt-daemon-driver-qemu libvirt-clients python3-libvirt
+        python3-pyyaml python3-flake8 \chromium-browser
+
+Updating the `node_modules` (in case you need to modify `package.json`)
+requires npm to be installed.
+
+    $ sudo apt-get install npm
+
+Additionally to build cockpit from the `autogen.sh` command below, head to /tools/debian/control 
```

Comment:
There are commands for both rpm and apt based distros to install build deps. For fedora it would be `sudo dnf builddep cockpit`

It's already mentioned here https://github.com/cockpit-project/cockpit/blame/main/HACKING.md#L35

You can exend that section to include the apt build-dep command.

Reply:
the `sudo apt-get builddep cockpit` does not work when i run it. What could be the issue?

Reply:
That depends on what exactly it ways when you run it. A common cause is that you don't have any `deb-src` apt sources.

Reply:
Im trying to run it from the root of the repo, it just shows that its unable to find a source package for tools/cockpit.spec

Reply:
Alright i got it running by enabling deb-src, should i mention about it?